src/components/pages/AboutPage.js

```diff
@@ -127,12 +127,23 @@ export default function AboutPage() {
             <Typography variant="h6">Madison Dong</Typography>
             <Typography variant="subtitle1">Student Developer</Typography>
             <Typography className={classes.blurbs}>
-              Madison Dong is a (soon-to-be) junior studying journalism,
-              computer science, and design,
-              so she thinks art and tech go well together.
-              Her three pillars of life are: adventures with friends,
+              I'm a junior studying journalism,
+              computer science, and design.
+              My three pillars of life are: adventures with friends,
               7-Eleven Big Bite hot dogs, and Frank Ocean.
-              Send her funny emails and/or articles: madisonfdong@gmail.com
+              Send me funny emails and/or articles: madisonfdong@gmail.com
+            </Typography>
+          </Paper>
+
+          <Paper elevation={2} className={classes.studentCard}>
+            <img src={albertaProfile} alt="Alberta Yoo" className={classes.profileImage} />
+            <Typography variant="h6">Alberta Yoo</Typography>
+            <Typography variant="subtitle1">Student Developer</Typography>
+            <Typography className={classes.blurbs}>
+              Hi, I'm Alberta! I'm a junior and currently
+              studying computer science. Enthusiastic about
+              the environment, love hot cheetos and 
+              just chillin 😎
```

Comment:
You're going to have to wrap this emoji in a `<span>`, check out Helen's profile for an example